stencils/RoutingState.stencil

```diff
@@ -65,3 +65,49 @@ class ObjectRoutingState<T: Equatable>: RoutingState {
         super.close(animated: animated, force: force)
     }
 }
+
+class IdentifiableRoutingState<ID : Equatable> : RoutingState {
```

Comment:
Can you conform to `Identifiable` protocol here?

Reply:
Improved the structure, so ID is now Identifiable